src/proxy/query_test.go

```diff
@@ -123,9 +124,11 @@ func TestProxyQueryComments(t *testing.T) {
 	}
 }
 
+// Proxy with no backup
 func TestProxyQueryStream(t *testing.T) {
 	log := xlog.NewStdLog(xlog.Level(xlog.PANIC))
 	fakedbs, proxy, cleanup := MockProxy(log)
+	// fakedbs, proxy, cleanup := MockProxyWithBackup(log)
```

Comment:
This line here is for?

Reply:
It is used to test proxyWithBackup, I forget to delete this line,  I have delete it just now.